src/libraries/Native/Unix/System.Native/pal_threading.h

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#pragma once
+
+#include "pal_compiler.h"
+#include "pal_types.h"
+
+typedef struct LowLevelMonitor LowLevelMonitor;
```

Comment:
The original had two layers. I have simplified it to just one layer and reformatted it to match the style used in the rest of System.Native.